lib/sqlalchemy/dialects/mysql/base.py

```diff
@@ -3512,7 +3519,7 @@ def __getitem__(self, index):
     def __getattr__(self, attr):
         item = getattr(self.rowproxy, attr)
         if isinstance(item, _array):
```

Comment:
We can probably just remove this if (and the above one), considering that no one reported the error and 3.9 was more than 4 years ago.